plugins/modules/zos_copy.py

```diff
@@ -1190,7 +1052,7 @@ def _mvs_copy_to_uss(
             else:
                 copy.copy_pds2uss(src, dest, is_binary=self.is_binary)
         except Exception as err:
-            self.fail_json(msg=str(err))
+            raise CopyOperationError(msg=str(err))
 
 
 class PDSECopyHandler(CopyHandler):
```

Comment:
At first I didn't want to change the interface in case it'd introduce bugs, but while doing some other changes I saw it was pretty straightforward to remove `dest_exists`, so now the handlers don't have it.